tasks/geonode.yml

```diff
@@ -73,16 +108,22 @@
        virtualenv={{virtualenv_dir}}/{{app_name}}
   when: github_user == "GeoNode"
 
+- name: install bcrypt
+  pip:
+    name: "django[bcrypt]"
+    virtualenv: "{{virtualenv_dir}}/{{app_name}}"
+
 - name: checkout latest web app code from geonode-project template
   shell: cd {{app_code_dir}} && {{virtualenv_dir}}/{{app_name}}/bin/django-admin.py startproject {{app_name}} --template=https://github.com/GeoNode/geonode-project/archive/master.zip -epy,rst,yml -n Vagrantfile
   when: github_user == "GeoNode" and (app_dir.stat.isdir is not defined)
 
 - name: copy the local_settings.py file in place
   template: src=local_settings.py.j2 dest={{app_code_dir}}/{{app_name}}/{{app_name}}/local_settings.py
 
-- name: patch the settings.py file in place
-  shell: sed -i -e "s/ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', \['localhost', \])/ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', \['localhost', '{{ server_name }}'\])/g" {{app_code_dir}}/{{app_name}}/{{app_name}}/settings.py
-  when: github_user != "GeoNode"
+# Just in local_settings now
```

Comment:
Can we add a separate flag for this, then?  You can default to true if you like.